verification/curator-service/ui/src/components/Case.tsx

```diff
@@ -72,7 +73,7 @@ export interface Travel {
         end: string;
     };
     purpose?: string;
-    method?: string;
+    methods?: string[];
```

Comment:
I didn't know there could be multiple methods for one travel object - just saw that in the schema. We need to update the NewCaseForm for that. (Mostly a note to self, don't need to add it in this PR)